Leetcode/Python/3_reverseNumber.py

```diff
@@ -0,0 +1,36 @@
+"""
+https://leetcode.com/problems/reverse-integer/
+
+Given a signed 32-bit integer x, return x with its digits reversed.
+If reversing x causes the value to go outside the signed 32-bit integer range [-231, 231 - 1], then return 0.
+
+"""
+
+
+class Solution:
+    def reverse(self, x: int) -> int:
+        reversed = 0
+
+        if x < -2147483648 or x > 2147483647 or x == 0:
```

Comment:
Magic numbers :- using hardcoded values within code is not preferred.
MIN_VALUE, MAX_VALUE are suggestions. would be good to define them at the top and use when needed